### PR TITLE
Consolidate all tslint configuration files into one

### PR DESCRIPTION
**Pull request by snario**
This standardizes our linting across all `.tsx?` files in the repo.

### CODE REVIEW

**Comment by joelalejandro** [comment]
Also, what about using non-js files for this, since it isn't actually code. The `.prettierrc` syntax we have on `web3torrent` / `embedded-wallet` looks nicer to implement and the `extends` rule mimics that of `tsconfig.json`.

**Comment by snario** [comment]
Okay @joelalejandro I have changed this branch so that now what it does it consolidates all the linting configurations but changes the absolute minimum source code. Thus, from here we can slowly remove all the special case rules each package has by simply removing the rule from the config file and running `yarn lint:write` on a per package basis.

File: packages/wallet/.prettierrc.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// NOTE: This package has inconsistent use of trailing commas and bracket spacing
+
+module.exports = {
+    ...require('../../.prettierrc.js'),
+    "printWidth": 120,
+    "singleQuote": false,
+    "bracketSpacing": false,
```
Comment (cf19drofxots) [inline packages/wallet/.prettierrc.js]: What's the reason for using these values?

I, for one, prefer a shorter width (80) and that we have `bracketSpacing` on.

**Comment by cf19drofxots** [comment]
> Thus, from here we can slowly remove all the special case rules each package has by simply removing the rule from the config file and running yarn lint:write on a per package basis.

Ignore my previous comment.

File: packages/wallet/.prettierrc.js (hunk 1 above)
Comment (snario) [reply packages/wallet/.prettierrc.js]: Me too; this is just how it is now.

**Pull request merged**